ports/rp2/machine_pin.c

```diff
@@ -515,8 +515,16 @@ static const mp_rom_map_elem_t machine_pin_locals_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_ALT_SIO), MP_ROM_INT(GPIO_FUNC_SIO) },
     { MP_ROM_QSTR(MP_QSTR_ALT_PIO0), MP_ROM_INT(GPIO_FUNC_PIO0) },
     { MP_ROM_QSTR(MP_QSTR_ALT_PIO1), MP_ROM_INT(GPIO_FUNC_PIO1) },
+    #if NUM_PIOS >= 3
+    { MP_ROM_QSTR(MP_QSTR_ALT_PIO2), MP_ROM_INT(GPIO_FUNC_PIO2) },
+    #endif
     { MP_ROM_QSTR(MP_QSTR_ALT_GPCK), MP_ROM_INT(GPIO_FUNC_GPCK) },
     { MP_ROM_QSTR(MP_QSTR_ALT_USB), MP_ROM_INT(GPIO_FUNC_USB) },
+    #if PICO_RP2350
+    { MP_ROM_QSTR(MP_QSTR_ALT_XIP_CS1), MP_ROM_INT(GPIO_FUNC_XIP_CS1) },
+    { MP_ROM_QSTR(MP_QSTR_ALT_CORESIGHT), MP_ROM_INT(GPIO_FUNC_CORESIGHT_TRACE) },
```

Comment:
Should this be `MP_QSTR_ALT_CORESIGHT_TRACE` to match both the C-level enum `GPIO_FUNC_CORESIGHT_TRACE` and the `af_fn` name?

Looks like there are 3 options for naming this alt func: `TRACE`, `CORESIGHT` and `CORESIGHT_TRACE`.  All 3 seem reasonable to me (the C enum is `GPIO_FUNC_CORESIGHT_TRACE` and the pin alt func names all start with TRACE).  But which ever one is chosen, the QSTR here should match it.

Reply:
I renamed the QSTR to `ALT_CORESIGHT_TRACE`.